### PR TITLE
TF-2683 Integrate update always read receipt setting in composer

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Issue
- #2615 
- #2683 

### Demo
**Web**

https://github.com/linagora/tmail-flutter/assets/160106668/d910d3ab-1356-4bc0-837b-e2b7283eae7c

**Mobile**

https://github.com/linagora/tmail-flutter/assets/160106668/734112a1-4a18-49b9-838c-d7101b66b12e

### CODE REVIEW

**Review by dab246 — COMMENTED** [review]
Please rebase

**Comment by hoangdat** [comment]
rebase and squash. Thanks

File: lib/features/composer/presentation/composer_controller.dart [hunk 1]
```diff
@@ -735,7 +735,7 @@ class ComposerController extends BaseController {
       },
       headerUserAgent: {IndividualHeaderIdentifier.headerUserAgent : userAgent},
       attachments: attachments.isNotEmpty ? attachments : null,
-      headerMdn: {IndividualHeaderIdentifier.headerMdn: getEmailAddressSender()},
+      headerMdn:hasRequestReadReceipt.value ? {IndividualHeaderIdentifier.headerMdn: getEmailAddressSender()} : {},
```
Comment (dab246) [inline lib/features/composer/presentation/composer_controller.dart:738]:
```suggestion
      headerMdn: hasRequestReadReceipt.value ? { IndividualHeaderIdentifier.headerMdn: getEmailAddressSender() } : {},
```

File: lib/features/composer/presentation/composer_bindings.dart [hunk 2]
```diff
@@ -186,6 +187,7 @@ class ComposerBindings extends BaseBindings {
     Get.lazyPut(() => DownloadImageAsBase64Interactor(Get.find<ComposerRepository>()));
     Get.lazyPut(() => TransformHtmlEmailContentInteractor(Get.find<EmailRepository>()));
     Get.lazyPut(() => GetAlwaysReadReceiptSettingInteractor(Get.find<ServerSettingsRepository>()));
+    Get.lazyPut(() => UpdateAlwaysReadReceiptSettingInteractor(Get.find<ServerSettingsRepository>()));
```
Comment (dab246) [inline lib/features/composer/presentation/composer_bindings.dart:190]: It is not used in `ComposerController`. Please delete it if not needed

Comment (tddang-linagora) [reply lib/features/composer/presentation/composer_bindings.dart]: Done

File: lib/features/composer/presentation/composer_controller.dart (hunk 1 above)
Comment (tddang-linagora) [reply lib/features/composer/presentation/composer_controller.dart]: Done

**Review by dab246 — APPROVED** [review]
Please rebase & squash

**Pull request merged**